CHANGELOG.md

```diff
@@ -107,6 +107,7 @@
 
 ## Improvements
 
+- PR #4331 Improved test for double that considers an epsilon
```

Comment:
This is still in 0.13 changelog

Reply:
I have addressed your concern. Can you please check?